README.md

```diff
@@ -80,5 +80,17 @@ Once CMake has generated the build files, within `<WorkSpace Dir>/build` run...
 - [Build] `cmake --build .` or `ninja -v` (if the build generator was Ninja)
 - [Test] `ctest --parallel`
 
+Documentation
+--------------
+**Latest release**
+
+-[0.3.6-preview.4](https://featurizerslibrarybuild.blob.core.windows.net/archive/0.3.6-preview.4/Documentation/Microsoft.Featurizer-cpp/v1.0/html/index.html)
+

```

Comment:
Now that I think about this, we probably only need 1 link for the most recent version of a release. Meaning, we only need 0.3.6-preview.4 for now. When 0.3.6 is officially released, we can remove the link to this preview version and only maintain a link to the official documentation.